test/functional/driver/driver-e2e-specs.js

```diff
@@ -83,30 +83,6 @@ describe('XCUITestDriver', function () {
         simsAfter.should.equal(simsBefore);
       });
 
-      it('with udid: uses sim and shuts it down afterwards', async () => {
```

Comment:
Sure, let's let @imurchie weigh in. Maybe we should get rid of this test after all if it's not asserting much.